### DISCUSSION TITLE
Specifying Faker parameters

### DISCUSSION THREAD

**Discussion by lbrenman** [stoplightio/prism#2087] [Q&A]
How do I specify Faker parameters in my schema?

For example, `x-faker: "address.cardinalDirection"` works but I'd like to pass the parameter useAbbr: boolean = false as per [https://fakerjs.dev/api/address.html#cardinalDirection.](https://fakerjs.dev/api/address.html#cardinalDirection).

Any advice?

If using the x-faker property is not the preferred way to control prism dynamic data behavior, please advise of the preferred method.

Thanks, in advance.

**Comment by yspotts** [comment]
The following worked for me

```
x-faker:
  datatype.number:
    min: 50
    max: 100
```

**Reply by lbrenman** [reply]
Thanks @yspotts, that's helpful. I appreciate it.

I tried several and some work and others don't :/

For example:

`total:
    type: string
    x-faker:
      commerce.price:
        min: 5
        max: 100
        dec: 2`

Results in `NaN`

and 

`account_number:
    type: string
    x-faker:
      finance.iban:
        formatted: true
        countryCode: 'DE'`

Results in `CH75 4556 3746 8T53 2719 8` instead of `DEXX XXXX XXXXX XXXX ...` as the documentation indicates. I tried with and without quotes around the Country Code and there was no difference.

Either not all parameters are supported or I'm still doing something wrong :/
